apps/music/index.html

```diff
@@ -98,8 +98,8 @@ <h1 id="title-text" data-l10n-id="music"></h1>
         <div id="player-cover">
           <div id="player-cover-image"></div>
           <div id="player-cover-caption">
-            <div id="player-cover-artist" data-l10n-id="songArtist">Song Artist</div>
-            <div id="player-cover-album" data-l10n-id="songAlbum">Song Album</div>
+            <div id="player-cover-artist"><bdi data-l10n-id="songArtist">Song Artist</bdi></div>
+            <div id="player-cover-album"><bdi data-l10n-id="songAlbum">Song Album</bdi></div>
```

Comment:
@hfiguiere open.html also has the player view in it, so be sure to make these html changes on it or we will break the open activity.